docs/get-started/developers/web/elements.md

```diff
@@ -1,33 +1,57 @@
-## Installation
+## Setup
 
-The Warp Elements package can be installed from NPM.
+Warp custom elements are supplied by the Eik CDN. In addition, you can install the @warp-ds/elements package to get intellisense in your editor.
+Add the following script and style to the very top of your document head to ensure performant loading:
 
-```shell
-npm install @warp-ds/elements
+```html
+<link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/elements/v2/styles.css" crossorigin />
+<script src="https://assets.finn.no/pkg/@warp-ds/elements/v2/index.js" type="module" crossorigin></script>
 ```
 
-## Setup
+If you are using `@borealis/layout-express` or `@borealis/layout-fastify` in your app these files may be provided for you. Check with the Web Platform Team in the #ask-web-platform slack channel.
 
-Since warp components are internationalised, you'd need to either set `NMP_LANGUAGE` for server-side rendering of the component, or set html document lang with a two-letter-code, eg: `<html lang="en">`. The server-side variable is read as `process.env.NMP_LANGUAGE` so you could either define this in your package.json script, or use `.env` files to leverage [dotenv](https://github.com/motdotla/dotenv).
+### Internationalisation
 
+Since warp components are internationalised, you need to ensure that you set the HTML document lang with a two-letter-code, eg: `<html lang="en">`.
```

Comment:
```suggestion
Since warp components are internationalised, ensure that you set the HTML document lang with a two-letter-code, eg: `<html lang="en">`.
```